styles/Hero.module.css

```diff
@@ -0,0 +1,36 @@
+/* styling for hero component */
+.hero {
+  display: flex;
+  justify-content: space-between;
+  padding: 64px 0px;
+  border-top: 1px solid var(--border-default);
+  border-bottom: 1px solid var(--border-default);
+  gap: 0px 64px;
+}
+
+.heroSection {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 50%;
+}
+
+.heroSectionSolgan {
+  color: var(--text-default);
+}
+
+.heroSectionTxt {
+  color: var(--text-default);
+  max-width: 70%;
+}
+
+.heroPhoto {
+  border-radius: 16px;
+  height: 100%;
+}
+
+.heroIcon {
```

Comment:
Camel case or kebab case for these class names?

Reply:
> Camel case or kebab case for these class names?

I fixed other problems you mentioned but still not too sure about this one.
Did you mean "Camel case should not be used here, use kebab case instead"

Reply:
Camelcase is working with no issue here as a class name. But I personally prefer the kebab case.

The reasons are:
- Css syntaxes are normally case-insensitive. Mixing with Camelcase can potentially cause some hidden problems.
- Kebab case is the same as CSS default syntax so easier to read.
- BEM uses kebab case and underscore and it is the most popular CSS naming convention.

However, camelcase is no technical issue here. So I think you can discuss with the team about this and decide later.